Userland/Libraries/LibMarkdown/Text.cpp

```diff
@@ -80,9 +92,25 @@ void Text::LinkNode::render_to_html(StringBuilder& builder) const
     }
 }
 
-void Text::LinkNode::render_for_terminal(StringBuilder&) const
+void Text::LinkNode::render_for_terminal(StringBuilder& builder) const
 {
-    // FIXME.
+    StringBuilder href_builder;
+    href->render_for_terminal(href_builder);
+    String href_string = href_builder.build();
+
+    bool is_linked = href_string.contains("://");
```

Comment:
Observation: This makes links to files non-clickable. An unusual decision, but fine by me.

Reply:
This was the previous behavior of LibMarkdown. I am open to suggested behavior.

Reply:
Still an unusual decision, but if that's the previous behavior then let's stick to it, you're right.